tests/CMakeLists.txt

```diff
@@ -41,8 +42,9 @@ if(AVIF_LOCAL_LIBGAV1)
     set_target_properties(avifyuv PROPERTIES LINKER_LANGUAGE "CXX")
 endif()
 target_link_libraries(avifyuv avif ${AVIF_PLATFORM_LIBRARIES})
-
-add_custom_target(avif_test_all COMMAND $<TARGET_FILE:aviftest> ${CMAKE_CURRENT_SOURCE_DIR}/data DEPENDS aviftest)
+foreach(AVIFYUV_MODE 0 2) # Modes 1 and 3 take more than 2 minutes each so they are disabled.
```

Comment:
Nit: To make the code more readable, we may want to use the names of these modes `limited`, `drift`, `rgb`, `premultiply` instead of 0, 1, 2, 3.

Reply:
Done at https://github.com/AOMediaCodec/libavif/pull/948